src/Sulu/Bundle/MediaBundle/Controller/CollectionController.php

```diff
@@ -40,7 +40,10 @@
 /**
  * Makes collections available through a REST API.
  */
-class CollectionController extends AbstractRestController implements ClassResourceInterface, SecuredControllerInterface, SecuredObjectControllerInterface
+class CollectionController extends AbstractRestController implements
+    ClassResourceInterface,
+    SecuredControllerInterface,
+    SecuredObjectControllerInterface
```

Comment:
```suggestion
class CollectionController extends AbstractRestController implements ClassResourceInterface, SecuredControllerInterface, SecuredObjectControllerInterface
```

Would keep this as it is as we currently not use this code style somewhere else.